python/sglang/srt/server_args.py

```diff
@@ -68,6 +69,10 @@ class ServerArgs:
     def __post_init__(self):
         if self.tokenizer_path is None:
             self.tokenizer_path = self.model_path
+        if not self.served_model_name:
+            self.served_model_name = self.model_path
+        elif isinstance(self.served_model_name, list):
+            self.served_model_name = self.served_model_name[0]
```

Comment:
If `served_model_name` is given in a wrong format (not a list), `self.served_model_name` will be `None`. This could trigger errors in other parts of the changes. Could you check?

Reply:
Sure, I'll check it.